src/js/experimental/media-captions-listbox.js

```diff
@@ -122,10 +122,35 @@ class MediaCaptionsListbox extends MediaChromeListbox {
   }
 
   #onChange() {
-    // turn off currently selected tracks
-    toggleSubsCaps(this, false);
+    const showingSubs = this.mediaSubtitlesShowing;
+    const showingSubsStr = this.getAttribute(
+      MediaUIAttributes.MEDIA_SUBTITLES_SHOWING
+    );
+    // NOTE: Since 'change' events and thus the onChange method will be fired/invoked
+    // from both user interaction and programmatic value changes, *including those
+    // propagated from the media controller itself*, we can check if the value (which
+    // will change from user interactions/direct programmatic changes) is actually different
+    // the mediaSubtitlesShowing value (which won't have been updated yet if the change was
+    // "local to the component") to determine if we need to request state changes. (CJP)
```

Comment:
Yeah it did a full round trip twice in these cases. Since the places with comments got missed by our core team, I want to include some comment in these places. Given your concerns, I'll make it brief:
`// Don't make request if this was the result of a media state state change`